src/scanner/label-in-name-utils.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/*
```

Comment:
@DaveTryon do you think this message is sufficient to satisfy the copy-left license agreement?

Reply:
After discussion with CELA, it is noted that these changes don't rise to the level of copy-lefting and attribution is covered in our NOTICE file.